src/bugsnag_tools/core.clj

```diff
@@ -15,14 +15,38 @@
     path
     (str host path)))
 
+(defn retry
+  "Given a function, f, run it after delay-ms milliseconds have
+  passed."
+  [f delay-ms]
+  (Thread/sleep delay-ms)
+  (f))
+
+(defn rate-limit-middleware
+  [client]
+  (fn [req]
+    (try
+      (client req)
+      (catch clojure.lang.ExceptionInfo e
+        (let [data (ex-data e)]
+          (if (= (:status data) 429)
+            (do
+              (println "limiting...")
```

Comment:
This is the only print performed by this library, is this something we need to bother sending to stdout?